Firestore/Swift/Source/Codable/FieldValue+Encodable.swift

```diff
@@ -49,6 +49,16 @@ public enum ServerTimestamp: Codable, Equatable {
   /// value to `stamp`.
   case resolved(Timestamp)
 
+  /// Get the `Optional` representation of `ServerTimestamp`.
```

Comment:
Similar concern about phrasing here. What do you think of this?

> Returns this value as an `Optional<Timestamp>`.
> 
> If the server timestamp is still pending, the returned optional will be `.none`. Once resolved, the returned optional will be `.some` with the resolved timestamp.

Reply:
Done.